### PR TITLE
Remove autocorrect in username box

### PR DESCRIPTION
**Pull request by CosmicWebServices**

### CODE REVIEW

**Comment by themightychris** [comment]
Addressed with 3dc482a and JarvusInnovations/emergence-skeleton-v2@832aad2aa2887e5b9cdc5210d7ed752e0f0326aa

File: html-templates/login/login.tpl [hunk 1]
```diff
@@ -26,7 +26,7 @@
         <input type="hidden" name="_LOGIN[return]" value="{refill field=_LOGIN.return default=$.server.REQUEST_URI}">
 
         <fieldset class="shrink">
-            {field inputName=_LOGIN[username] label=Username hint='You can also log in with your email address.' required=true autofocus=true}
+            {field spellcheck='false' inputName=_LOGIN[username] label=Username hint='You can also log in with your email address.' required=true autofocus=true}
```
Comment (themightychris) [inline html-templates/login/login.tpl:29]: adding `spellcheck` here will not change the output HTML without adding support for that parameter to the `{field}` template's definition

**Comment by CosmicWebServices** [comment]
Good

**Review by themightychris — COMMENTED** [review]
changes made upstream and merged

**Pull request closed without merging**